### PR TITLE
[WIP] [NONE] Update Contributor Guide

### PR DESCRIPTION
**Pull request by NicoPrediger**
[NONE] Update Contributor Guide

Signed-off-by: Nico Prediger <mail@nicoprediger.de>

### CODE REVIEW

**Comment by NicoPrediger** [comment]
Planning on adding info about runtime configuration later...

**Review by miklossy — APPROVED** [review]

**Comment by laeubi** [comment]
@miklossy as there are some non trivial steps to do to setup an IDE, you might can take a look at https://github.com/eclipse/gef/issues/86 to provide an Eclipse-Installer profile? That would ease installation for new users.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,116 +1,132 @@
...
 
 ## Run a headless build
+
 GEF uses a Maven/Tycho-based build infrastructure. With the [Eclipse Maven Integration](http://eclipse.org/m2e/), a headless build can also be executed in the local workspace. Make sure you have checked out all projects of the respective git repository. Then easily run the build by right-clicking the `pom.xml` file located within the `org.eclipse.gef.releng` project, and selecting 'Run As -> Maven build...'.
 
-In the configuration wizard you have to specify the goals ("clean verify") and the target profile (e.g. "NEON.target") for the build. Note that without setting the target profile the build will not run successfully. The build needs to be executed with a Java 8 VM, so if you have installed multiple JVMs, you should check the settings in the *JRE* tab.
+In the configuration wizard you have to specify the goals ("clean verify") and the target profile (e.g. "2019-03.target") for the build. Note that without setting the target profile the build will not run successfully. The build needs to be executed with a Java 11 VM, so if you have installed multiple JVMs, you should check the settings in the *JRE* tab.
```
Comment (laeubi) [inline CONTRIBUTING.md:103]: It seems one has also to define 'JAVAFX_HOME' env variable.
Also the build complains about missing 'javadoc' variable

Comment (miklossy) [reply CONTRIBUTING.md:103]: @laeubi: Thanks for the hints and for the feedback!

**Comment by nyssen** [comment]
Correct. We had to add this when adopting Java 11 and as it seems, we forgot to update the contributor guide. Thanks for pointing that out.

Best Regards,
Alexander

> Am 19.03.2020 um 08:48 schrieb Christoph Läubrich <notifications@github.com>:
> 
> 'JAVAFX_HOME' env variable.

**Comment by miklossy** [comment]
@Hejado @nyssen : What is the state of this PR? Is it ready to be merged?

**Comment by nyssen** [comment]
@Hejado wanted to cross-check when setting things up on his machine. From my point of view, nothing is open here.

**Comment by miklossy** [comment]
@nyssen Thanks for clarifying it, I will merge this PR right now.
@Hejado for further improvements please open follow up PRs.

**Pull request merged**